pkg/middleware/util.go

```diff
@@ -25,6 +28,8 @@ func Gziper() macaron.Handler {
 			return
 		}
 
-		ctx.Invoke(macaronGziper)
+		if _, err := ctx.Invoke(gziper); err != nil {
+			gziperLogger.Debug("Invoking gzip handler failed", "err", err)
```

Comment:
This should probably be logged as an error, don't you think?

Reply:
@marefr I wasn't sure, but if you say so I guess it should :)